src/components/avatar/theme.css

```diff
@@ -185,6 +184,7 @@
     color: white;
     text-align: center;
     text-transform: uppercase;
+    letter-spacing: 0px;
```

Comment:
Q: is this rule really needed? Won't the browser default `normal` value already do the trick?

Reply:
A: This rule can idd be dropped

Reply:
The Heading4 has a letter-spacing of .6px by default so it needs to be "reset" manually in the css.